conda_smithy/cli.py

```diff
@@ -192,16 +192,52 @@ def __init__(self, parser):
         scp = self.subcommand_parser
         scp.add_argument("--feedstock_directory", default=os.getcwd(),
                          help="The directory of the feedstock git repository.")
+        scp.add_argument("-c", "--commit", nargs='?', choices=["edit", "auto"], const="edit",
+                         help="Whether to setup a commit or not.")
 
     def __call__(self, args):
         try:
             configure_feedstock.main(args.feedstock_directory)
-            print("\nCI support files regenerated. These need to be pushed to github!")
-            print("\nYou can commit the changes with:\n\n"
-                  "    git add -A\n"
-                  "    git commit -m 'MNT: Re-rendered with conda-smithy %s'" % __version__)
+            print("\nRe-rendered with conda-smithy %s'." % __version__)
```

Comment:
Missed that there was still a `'` in this string. Removing it in PR ( https://github.com/conda-forge/conda-smithy/pull/342 ).